README.md

```diff
@@ -1,3 +1,10 @@
 # Brain AutoML
 
-This repository contains a list of AutoML related models and libraries.
+This repository contains a list of all the necessary  AutoML related models and libraries.
```

Comment:
You have added extra space in line 3.

Reply:
Thankyou for your review I will correct that